assets/xml/objects/object_fr.xml

```diff
@@ -1,67 +1,68 @@
+        <DList Name="gFrogLeftHandDL" Offset="0x9AE0" />
+        <DList Name="gFrogLeftUpperArmDL" Offset="0x9B80" />
+        <DList Name="gFrogRightForearmDL" Offset="0x9D88" />
+        <DList Name="gFrogRightHandDL" Offset="0x9FB0" />
+        <DList Name="gFrogRightUpperArmDL" Offset="0xA050" />
+        <DList Name="gFrogLowerBodyDL" Offset="0xA258" />
+        <DList Name="gFrogRightHeelDL" Offset="0xA428" />
+        <DList Name="gFrogRightToeDL" Offset="0xA6B8" />
+        <DList Name="gFrogRightShinDL" Offset="0xA758" />
+        <DList Name="gFrogRightThighDL" Offset="0xA978" />
+        <DList Name="gFrogLeftHeelDL" Offset="0xABA8" />
+        <DList Name="gFrogLeftToeDL" Offset="0xAE38" />
+        <DList Name="gFrogLeftShinDL" Offset="0xAED8" />
+        <DList Name="gFrogLeftThighDL" Offset="0xB0F8" />
+        <DList Name="gFrogDustMaterialDL" Offset="0xB328" />
+        <DList Name="gFrogDustModelDL" Offset="0xB338" />
+        <Limb Name="gFrogLowerBodyLimb" Type="Standard" EnumName="FROG_LIMB_LOWER_BODY" Offset="0xB3C8" />
+        <Limb Name="gFrogUpperBodyLimb" Type="Standard" EnumName="FROG_LIMB_UPPER_BODY" Offset="0xB3D4" />
+        <Limb Name="gFrogRootLimb" Type="Standard" EnumName="FROG_LIMB_ROOT" Offset="0xB3E0" />
+        <Limb Name="gFrogHeadLimb" Type="Standard" EnumName="FROG_LIMB_HEAD" Offset="0xB3EC" />
+        <Limb Name="gFrogRightIrisLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_IRIS" Offset="0xB3F8" />
+        <Limb Name="gFrogJawLimb" Type="Standard" EnumName="FROG_LIMB_JAW" Offset="0xB404" />
+        <Limb Name="gFrogRightEyeLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_EYE" Offset="0xB410" />
+        <Limb Name="gFrogLeftEyeLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_EYE" Offset="0xB41C" />
+        <Limb Name="gFrogLeftIrisLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_IRIS" Offset="0xB428" />
+        <Limb Name="gFrogLeftUpperArmLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_UPPER_ARM" Offset="0xB434" />
+        <Limb Name="gFrogLeftForearmLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_FOREARM" Offset="0xB440" />
+        <Limb Name="gFrogLeftHandLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_HAND" Offset="0xB44C" />
+        <Limb Name="gFrogRightUpperArmLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_UPPER_ARM" Offset="0xB458" />
+        <Limb Name="gFrogRightForearmLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_FOREARM" Offset="0xB464" />
+        <Limb Name="gFrogRightHandLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_HAND" Offset="0xB470" />
+        <Limb Name="gFrogRightThighLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_THIGH" Offset="0xB47C" />
+        <Limb Name="gFrogRightShinLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_SHIN" Offset="0xB488" />
+        <Limb Name="gFrogRightHeelLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_HEEL" Offset="0xB494" />
+        <Limb Name="gFrogRightToeLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_TOE" Offset="0xB4A0" />
```

Comment:
Looking at Z64Utils, what do you think about `gFrogRightToesLimb`? Specifically, I made "toes" plural, since it looks like this limb renders four of them:
![image](https://github.com/zeldaret/mm/assets/12245827/a9bef35d-7d6d-43f1-a629-740307727928)

Reply:
oh yea i was thinking of using toes instead of toe but i thought writing in plural could be a bad idea but yeah toes is better